migrations/v5.js

```diff
@@ -0,0 +1,127 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, getCourse, getComponents, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('adapt-contrib-accordion - v4.0.0 > v5.0.0', async () => {
+  let course, courseAccordionGlobals, accordions;
+  const originalAriaRegion = 'Accordion. Select each button to expand the content.';
+  const newAriaRegion = 'List of expandable sections. Select each button to expand the content.';
+
+  whereFromPlugin('adapt-contrib-accordion - from v4.0.0', { name: 'adapt-contrib-accordion', version: '<5.0.0' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = getComponents('accordion');
+    return accordions.length;
+  });
+
+  mutateContent('adapt-contrib-accordion - modify globals ariaRegion attribute', async (content) => {
+    course = getCourse();
+    if (!_.has(course, '_globals._components._accordion.ariaRegion')) _.set(course, '_globals._components._accordion', { ariaRegion: originalAriaRegion });
```

Comment:
```suggestion
    if (!_.has(course, '_globals._components._accordion.ariaRegion')) {
      _.set(course, '_globals._components._accordion.ariaRegion', originalAriaRegion);
    }
```
Weird mix of property detection and value setting.

Reply:
@oliverfoster One tweak, if ariaRegion is missing set it to newAriaRegion instead.